requre/online_replacing.py

```diff
@@ -81,3 +84,38 @@ def _internal(*args, **kwargs):
         return _internal
 
     return decorator_cover
+
+
+def store(what: str, in_module: str):
```

Comment:
please make sure that "what" argument is precisely document so people don't have a "what" face :)

Reply:
It is just a quick wrapper on top of the `replace` decorator. We need to have it consistent between these two.

- Some suggestion for both places?